alea/model.py

```diff
@@ -95,19 +95,16 @@ def __init__(
         self._confidence_interval_kind = confidence_interval_kind
         self.confidence_interval_threshold = confidence_interval_threshold
         self.asymptotic_dof = asymptotic_dof
-        nominal_values = kwargs.get("nominal_values", {})
-        self._define_parameters(parameter_definition, nominal_values)
+        self._define_parameters(parameter_definition)
 
         self._check_ll_and_generate_data_signature()
+        self.set_nominal_values(overwrite_static=True, **kwargs.get("nominal_values", {}))
 
-    def _define_parameters(self, parameter_definition, nominal_values=None):
+    def _define_parameters(self, parameter_definition):
```

Comment:
I think `_define_parameters` should be as simple as just initializing `Parameters`.